tasks/knative.py

```diff
@@ -0,0 +1,286 @@
+import os
```

Comment:
Copy from the existing task in `faasm-experiments` with the up-to-date dependencies.